src/github.com/getlantern/flashlight/autoupdate/autoupdate.go

```diff
@@ -90,7 +100,7 @@ func applyNext() {
 	if httpClient != nil {
 		err := autoupdate.ApplyNext(&autoupdate.Config{
 			CurrentVersion: Version,
-			URL:            serviceURL,
+			URL:            updateServerURL + "/update",
```

Comment:
It still seems like it would be better to control all of that on the server side, though, since then we could make it whatever we want always. It's a really minor point, though -- merging!